### PR TITLE
Drop Pulp 2 repositories from katello-repos

### PR DESCRIPTION
**Pull request by ekohl**
Now that Pulp 2 is removed, there's no reason to ship these anymore.

### CODE REVIEW

**Comment by ekohl** [comment]
@evgeni is the qpid copr still needed or were the packages introduced in EPEL again?

**Comment by evgeni** [comment]
le copr can be gone, epel is good.

File: packages/katello/katello-repos/katello-repos.spec [hunk 1]
```diff
@@ -1,20 +1,11 @@
...
 %define repo_dir %{_sysconfdir}/yum.repos.d
 %define repo_dist %{dist}
 
 %global prereleasesource nightly
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
-%global release 3
+%global release 4
 
 Name:           katello-repos
 Version:        4.0
```
Comment (evgeni) [inline packages/katello/katello-repos/katello-repos.spec:11]: shouldn't nightly be at 4.1 now?

**Review by evgeni — APPROVED** [review]

File: packages/katello/katello-repos/katello-repos.spec (hunk 1 above)
Comment (ekohl) [reply packages/katello/katello-repos/katello-repos.spec:11]: Can I blame a lack of branching procedures in packaging? Looks like there's been an rc release in the 2.4 branch so I'll first submit a PR to bump to 4.1 and then rebase this.

**Comment by ekohl** [comment]
I opened https://github.com/theforeman/foreman-packaging/pull/6370 which I think is a replacement for this since it includes the change.

**Pull request closed without merging**